docs/build/dev-workflow/personas.md

```diff
@@ -31,11 +31,11 @@ Given this background, we can identify the following developer personas for Poly
 
 :::
 
-The following sections will give a description of the workflow for all of the above developer personas.
+The following sections will give a description of the workflow for all of the above developer personas, starting with application developers... .
```

Comment:
This is a bit confusing. Starts with application developer but then on the next line mentioned Infrastructure Operators